src/containers/PipelineRun/PipelineRun.js

```diff
@@ -261,6 +274,7 @@ export /* istanbul ignore next */ class PipelineRunContainer extends Component {
 
     const logContainer = (
       <Log
+        key={stepName}
```

Comment:
:sweat_smile: :sweat_smile:

I can tackle it separately if you like but If I missed it here I probably missed it in the other containers